build-tools/packages/build-cli/src/commands/check/policy.ts

```diff
@@ -273,7 +273,7 @@ export class CheckPolicy extends BaseCommand<typeof CheckPolicy> {
 					// Resolvers are expected to be run serially to avoid any conflicts.
 					// eslint-disable-next-line no-await-in-loop
 					const resolveResult = await runWithPerf(handler.name, "resolve", async () =>
-						resolver(relPath, gitRoot),
+						resolver(relPath, gitRoot, result),
```

Comment:
I'm not sure about this. If we want to provide better coordination between handlers and resolvers, IMO we should refactor it so that's more obvious. Seems like passing a string array around won't work for long.